csharp/ql/src/codeql-suites/csharp-lgtm-full.qls

```diff
@@ -7,10 +7,7 @@
     tags contain:
       - ide-contextual-queries/local-definitions
       - ide-contextual-queries/local-references
-- query: Metrics/Dependencies/ExternalDependencies.ql
-- query: Metrics/Dependencies/ExternalDependenciesSourceLinks.ql
```

Comment:
Should these two queries also be deprecated?

Reply:
That's a good idea. Done.

Reply:
It would also be nice to deprecate all public predicate/classes in `ExternalDependencies.qll` and `CodeDuplication.qll`, but I can also do that in a follow-up PR.

Reply:
Yes, that should be a follow-up PR.